requirements.txt

```diff
@@ -0,0 +1,10 @@
+fastapi==0.115.12
+uvicorn==0.30.6
+sqlalchemy==2.0.39
+asyncpg==0.29.0
+pydantic==2.11.0a2
+ruff==0.10.0
+black==25.1.0
+pytest==7.4.0
+httpx==0.28.1
+dotenv==0.9.9
```

Comment:
to chyba mocno stara wersja wydaje mi sie ze masz tu na mysli python-dotenv? ale tak czy siak mozna zmienic na python-dotenv>=1.0.0 imo ale to sprawdz